src/main/java/uk/gov/hmcts/dts/fact/controllers/SearchController.java

```diff
@@ -83,10 +83,13 @@ public ResponseEntity<List<CourtReferenceWithDistance>> findCourtsByPostcode(
     @SuppressWarnings("PMD.UseObjectForClearerAPI")
     public ResponseEntity<ServiceAreaWithCourtReferencesWithDistance> findCourtsByPostcodeAndServiceArea(
         @RequestParam Optional<String> postcode,
-        @ApiParam("Service Area Slug") @RequestParam(name = "serviceArea") Optional<String> serviceAreaSlug
+        @ApiParam("Service Area Slug") @RequestParam(name = "serviceArea") Optional<String> serviceAreaSlug,
+        @RequestParam Optional<String> action
```

Comment:
Will the action always be optional? I would have assumed with our three paths that this value would always be nearest, documents, updated, or not-listed? Perhaps an enumerator would also be good for this

If we change it to that, it will be easier to update this in the future. A little bit more work though, but more learning too! 

And for now we are only interested in adding the one condition around the action being nearest, so it would be more so for future maintenance